examples/plotter.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torch.nn.functional as F
+from torch.cuda import amp
+import pytest
+from tqdm import tqdm
+import torchvision.transforms as T
+from pytorch_cnn_trainer import dataset
+from pytorch_cnn_trainer import model_factory
+from pytorch_cnn_trainer import utils
+from pytorch_cnn_trainer import engine
+from torch.optim.swa_utils import SWALR
```

Comment:
May I know why so many imports are needed?

Can this file be in the library instead fo `examples` into `pytorch_cnn_trainer`?

Reply:
Actually I too was going to remove those imports but then forgot about it , , ,will do